nextcord/types/allowed_mentions.py

```diff
@@ -0,0 +1,78 @@
+"""
+The MIT License (MIT)
+Copyright (c) 2021-present vcokltfre & tag-epic
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
```

Comment:
As done in a recent update we use comments for the license instead of docstrings